example/verifyartifact/main.go

```diff
@@ -109,7 +109,6 @@ func main() {
 		}
 
 		err := runVerification(sev, pb, b)
-
```

Comment:
Could you please be so kind as to revert this unrelated change? It makes the review process more difficult.

You can use these commands:
```sh
git restore --source=master example/verifyartifact/main.go
git commit -a -m "Revert unrelated changes"
```

Thank you.

Reply:
Done